File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        piles.sort()
        rounds = len(piles) // 3
        idx = rounds
        result = 0
        
        for _ in range(rounds):
            result += piles[idx]
            idx += 2
        
        return result

File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.js
/**
 * @param {number[]} piles
 * @return {number}
 */
var maxCoins = function(piles) {
    piles.sort((a, b) => a - b);
    let totalCoins = 0;
    let rounds = Math.floor(piles.length / 3);
    let idx = rounds;
    
    for (let i = 0; i < rounds; i += 1){
        totalCoins += piles[idx];
        idx += 2;
    }
    
    return totalCoins;
};